freedomofpress/securedrop
# Local rsync fixes

## Status

:construction: - Waiting for CI to pass

## Description of Changes

Fixes #1722 and addresses an issue with the `synchronize.py` module that was brought in to fix docker 

## Testing

How should the reviewer test this PR?
* Ensure `vagrant up build` is working locally. As well as the full local staging tests doesnt break. 
* Make sure the CI tests have a green :white_check_mark: 
* Check to make sure you showered and brushed your teeth today. Did you remember to call Nana!!? Its her birthday!! COME ON!!

## Deployment

This relies on ansible 2.2+ and we are still working on getting #1146 into develop.

## Checklist

### If you made changes to the app code:

N/A

### If you made changes to the system configuration:

N/A
